services/gundeck/test/integration/Metrics.hs

```diff
@@ -0,0 +1,29 @@
+module Metrics where
+
+import Imports
+import TestSetup
+import Bilge
+import Bilge.Assert
+import Test.Tasty
+import Test.Tasty.HUnit
+import Control.Lens ((^.))
+
+type TestSignature a = Gundeck -> Http a
+
+test :: IO TestSetup -> TestName -> (TestSignature a) -> TestTree
+test setup n h = testCase n runTest
+  where
+    runTest = do
+        s <- setup
+        void $ runHttpT (s ^. tsManager) (h (s ^. tsGundeck))
+
+tests :: IO TestSetup -> TestTree
+tests s = testGroup "Metrics" [test s "prometheus" testPrometheusMetrics]
```

Comment:
Since gundeck and cannon are very closely related, and cannon doesn't have its own integration test suite, you could also add a test here for cannon.